src/network/networkd-dhcp6.h

```diff
@@ -4,6 +4,16 @@
 #include "sd-dhcp6-client.h"
 
 #include "conf-parser.h"
+#include "macro.h"
+
+typedef enum DHCP6ClientStartMode {
+        DHCP6_CLIENT_START_MODE_NO,
+        DHCP6_CLIENT_START_MODE_INFORMATION_REQUEST,
+        DHCP6_CLIENT_START_MODE_SOLICIT,
+        DHCP6_CLIENT_START_MODE_YES,
```

Comment:
It is not necessary to support boolean `yes`, as `WithoutRA=` is not included any released version.